### PR TITLE
Release workflow: Move inline bash to dedicated file

### PR DESCRIPTION
**Pull request by Malax**
As suggested by @edmorley here: https://github.com/heroku/buildpacks-jvm/pull/11#pullrequestreview-568462155

### CODE REVIEW

File: .github/scripts/release-workflow-package-push.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# This script is an integral part of the release workflow: .github/workflows/release.yml
+# It requires the following environment variables to function correctly:
+#
+# REQUESTED_BUILDPACK_ID - The ID of the buildpack to package and push to the container registry.
```
Comment (edmorley) [inline .github/scripts/release-workflow-package-push.sh:7]: If the env var is required, would it be worth checking for it explicitly and erroring out if not set?

Or alternatively, passing in as `$1` (and then assigning to a var explicitly here), so it's clearer in the GHA workflow file what is using `REQUESTED_BUILDPACK_ID`?

File: .github/scripts/release-workflow-package-push.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```
Comment (edmorley) [inline .github/scripts/release-workflow-package-push.sh:1]: Looking at the raw diff (https://patch-diff.githubusercontent.com/raw/heroku/buildpacks-jvm/pull/12.diff) the file has perms 644 - might need to be updated to 744 depending on how the workflow bash runner works.

**Review by edmorley — APPROVED** [review]

File: .github/scripts/release-workflow-package-push.sh (hunk 2 above)
Comment (Malax) [reply .github/scripts/release-workflow-package-push.sh:1]: Thank you! Good catch!

File: .github/scripts/release-workflow-package-push.sh (hunk 1 above)
Comment (Malax) [reply .github/scripts/release-workflow-package-push.sh:7]: I'll make the dependency more explicit. However, this also creates variables via echo that other steps need - it's a hairy ball of dependencies nonetheless. :)

**Pull request merged**